docs/content/users/configuration/config.md

```diff
@@ -103,6 +103,16 @@ Extra Debian packages for the project’s database container.
 
 Example: `dbimage_extra_packages: ["less"]` will add the `less` package when the database container is built.
 
+## `ddev_version_constraint`
+
+You can configure a [ddev version constraint](https://github.com/Masterminds/semver#checking-version-constraints) that will be validated against the running ddev and prevent any command from running if it doesn't match.
```

Comment:
Always capitalize DDEV when it’s used as a product name, vs. `ddev` (in backticks) referencing the command. Also added a word attempting to distinguish the two “DDEV” instances in this sentence—which ideally would be pared down if possible.

```suggestion
You can configure a [DDEV version constraint](https://github.com/Masterminds/semver#checking-version-constraints) that will be validated against the running DDEV executable and prevent any command from running if it doesn't match.
```